package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vuefire",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
I think `1.1.0` is a more correct version number and what we've been doing with the other *Fire repos. Users won't need to change any VueFire code, only Firebase code. Also, if we allow VueFire to work with the Firebase `2.x.x` SDK (see below), then this definitely won't be a breaking change in any way.